.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "nifxml"]
 	path = nifxml
-	url = https://github.com/niftools/nifxml.git
+	url = https://github.com/aperture-it/nifxml.git
```

Comment:
This is a temporary change that I need for synchronize the latest changes nifxml to the local repository.